CODEOWNERS

```
@@ -0,0 +1 @@
+* @molawson @sheck @danielma
```

In light of the recent creation of a "Balto Maintainers" team:

```suggestion
* @planningcenter/balto-maintainers
```

---

Yep... This looks like the route to go 👆🏼